Tweeter.Tests/DAL/TweeterRepositoryTests.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Tweeter.DAL;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Tweeter.Models;
+using Moq;
+using System.Data.Entity;
+
+namespace Tweeter.Tests.DAL
+{
+    [TestClass]
+    public class TweeterRepositoryTests
+    {
+        
+        private Mock<DbSet<ApplicationUser>> mock_users { get; set; }
+        private Mock<TweeterContext> mock_context { get; set; }
+        private TweeterRepository Repo { get; set; }
+        private List<ApplicationUser> users { get; set; }
+
+        [TestInitialize]
+        public void Initialize()
+        {
+            mock_context = new Mock<TweeterContext>();
+            mock_users = new Mock<DbSet<ApplicationUser>>();
+            Repo = new TweeterRepository(mock_context.Object);
+        }
+
+        public void ConnectToDatastore()
+        {
+            users = new List<ApplicationUser>();
+            var query_users = users.AsQueryable();
+
+            mock_users.As<IQueryable<ApplicationUser>>().Setup(m => m.Provider).Returns(query_users.Provider);
+            mock_users.As<IQueryable<ApplicationUser>>().Setup(m => m.Expression).Returns(query_users.Expression);
+            mock_users.As<IQueryable<ApplicationUser>>().Setup(m => m.ElementType).Returns(query_users.ElementType);
+            mock_users.As<IQueryable<ApplicationUser>>().Setup(m => m.GetEnumerator()).Returns(() =>query_users.GetEnumerator());
+
+            mock_context.Setup(c => c.TweeterUsers).Returns(/*Tweeter User List*/);
```

Comment:
Isn't there a [line in TweeterRepoTests.cs](https://github.com/nss-evening-cohort-3/tweeter/pull/23/commits/742cc0d8eb516cd0310468a8a5862dbbf58633fb#diff-d52f97db2404fd0c77b2c9142f6a8150R51) that shows you the argument for the `Returns` method?